server/bin/timely-standalone.sh

```diff
@@ -25,12 +25,15 @@ if [[ -e ${NATIVE_DIR} ]]; then
 fi
 mkdir -p ${NATIVE_DIR}
 
-$JAVA_HOME/bin/jar xf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so
+pushd ${BASE_DIR}/bin
+echo "$JAVA_HOME/bin/jar xvf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so"
+$JAVA_HOME/bin/jar xvf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so
+popd
 
-export CLASSPATH="${CONF_DIR}:${LIB_DIR}/*"
+export CLASSPATH="${CONF_DIR}:${LIB_DIR}/*:${LIB_DIR}/../bin/META-INF/native"
```

Comment:
Probably not. This was probably me desperately trying to get things running. I can remove and test that everything is ok.